sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -42,10 +41,10 @@ export interface AppConfigurationClientOptions {
 }
 
 // @public
-export interface ConfigurationSetting extends ConfigurationSettingParam {
+export type ConfigurationSetting<T = string> = ConfigurationSettingParam<T> & {
```

Comment:
Unsure of docs.microsoft.com.

"externds" is not rendered in github.io docs.(just plain text)
![image](https://user-images.githubusercontent.com/10452642/118320613-5a2c5d00-b4b1-11eb-9094-b8b5fa80b521.png)

Reply:
> What do you think about this? This at least gives you the actual range of types.

Feels like very little value, but can add. Just wondering about the too much text and confusion for the users.

Reply:
Added your suggestion anyway.

Reply:
One way to think about this - can 'T' really be any type I can imagine? Or is it constrained? Looking at it you can see it's not intended to be generic over everything, there are a few types that it can handle.